styles/output/webview-generic.css

```diff
@@ -2012,6 +2012,22 @@ blockquote {
   color: #006767;
 }
 
+.hljs-ln-numbers {
+  font-size: font-scale(1);
```

Comment:
i wonder why `font-scale(1)` isn't being converted to `1.2rem` like it is in pdf? 🤔 if you have to, feel free to hardcode the value instead of using `font-scale`

Reply:
not sure why this is happening, thus, hard coded the font-size value.